app/controllers/view_components_controller.rb

```diff
@@ -2,12 +2,14 @@
 
 require "rails/application_controller"
 
-class ViewComponentsController < Rails::ApplicationController # :nodoc:
+class ViewComponentsController < ActionController::Base # :nodoc:
```

Comment:
I spent some time yesterday looking into this as well and this line fixed the failing test. However, it made another test fail. Do you know why the inheritance matters here?

Reply:
The other failures in this branch seem to be related to the cache_classes configuration change. I wasn't able to dynamically update that config at runtime and make it work as expected.

I think the actual issue is that rails/application_controller is not reloaded by zeitwerk, perhaps because we require it manually and/or we don't have it present in the autoload_paths. BTW I just noticed we should remove the require line as well.